python/ray/actor.py

```diff
@@ -534,6 +534,73 @@ def options(self, **actor_options):
         The arguments are the same as those that can be passed
         to :obj:`ray.remote`.
 
+        Args:
```

Comment:
is it possible to add them and generate with option_utils? 
For example https://github.com/ray-project/ray/blob/master/python/ray/_private/ray_option_utils.py

we have task only options, actor only options, common options, I feel maybe the comment should be put there.

Reply:
Yes, that could be a long-term solution, but it does have a cost: developers wouldn't get a direct doc by looking at `options`. Maybe we should generate the doc automatically with a script and checks if the doc is update-to-date with a test.